apis/cache/v1alpha2/replication_group_types.go

```diff
@@ -316,61 +314,6 @@ type ReplicationGroup struct {
 	Status ReplicationGroupStatus `json:"status,omitempty"`
 }
 
-// SetBindingPhase of this ReplicationGroup.
-func (rg *ReplicationGroup) SetBindingPhase(p runtimev1alpha1.BindingPhase) {
-	rg.Status.SetBindingPhase(p)
-}
-
-// SetConditions of this ReplicationGroup.
-func (rg *ReplicationGroup) SetConditions(c ...runtimev1alpha1.Condition) {
```

Comment:
You have probably addressed this elsewhere, so apologize if I am repeating the same question:

How do we customize any of these methods? I would imagine if all of these methods are similar for different types, one could move them all to crossplane/run-time library and use reflection.

Reply:
I'm not sure I understand the question. Are you wondering what we'd do if we needed a slightly different implementation of one of these methods? The code generation tool won't replace any method that already exists, so it's safe to implement our own custom variants.

If you're wondering how we update the methods that are generated by the code generation tool, https://github.com/crossplaneio/crossplane-tools/blob/master/internal/method/method.go is the place to look.